src/utils/find.js

```diff
@@ -0,0 +1,13 @@
+/* the array.prototype.find polyfill on npmjs.com is ~20kb (not worth it)
+ * and lodash is ~200kb (not worth it)
+ */
+
+export function find(arr, func) {
```

Comment:
IE11 does not support Array.prototype.find. https://www.npmjs.com/package/array.prototype.find is 20kb so not worth it, and lodash's find would be perfect but I didn't want to bring in lodash's ~100kb just to polyfill Array.prototype.find